.travis.yml

```diff
@@ -2,6 +2,7 @@ language: node_js
 node_js:
   - "0.10"
   - "0.8"
+  - "io.js"
```

Comment:
ok, 1.0 might not be the best idea considering that 1.1 is not a major but a minor bump. If any, `iojs-1` would be better.